mkosi/config.py

```diff
@@ -1677,6 +1678,18 @@ def parse_ini(path: Path, only_sections: Collection[str] = ()) -> Iterator[tuple
         parse=config_make_list_parser(delimiter=" ", unescape=True),
         help="Set an environment variable when running scripts",
     ),
+    MkosiConfigSetting(
+        dest="environment_files",
+        long="--environment-files",
+        metavar="PATH",
+        name="EnvironmentFiles",
+        section="Content",
+        parse=config_make_list_parser(delimiter=",", parse=make_path_parser()),
+        paths=("mkosi.env",),
+        path_default=False,
+        help="Enviroment files to set when running scripts",
+        compat_names=("EnvironmentFile",),
```

Comment:
Please drop this as well, it's a new option, there are no compatibility names for this option.